client/components/contributepage/InfoSection.js

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import styled from "styled-components";
+
+const Article = styled.article`
+  display: grid;
+  place-items: center center;
+  line-height: 24px;
+  a {
+    color: blue;
```

Comment:
It would be better to define a new color in the ThemeProvider that we can use for all our a-tags - ex. "link: blue". This would make it easier to swap our colors at a later time 🙂

Reply:
:100: Will do, I wasn't sure if you all wanted us changing files outside the scope of the files we're on, but this makes more sense

Reply:
As a general rule, I don't like changes outside of the files you work on - however adding colors etc. is fine :)